docs/reference/mapping/types/sparse-vector.asciidoc

```diff
@@ -26,10 +26,70 @@ PUT my-index
 
 See <<semantic-search-elser, semantic search with ELSER>> for a complete example on adding documents to a `sparse_vector` mapped field using ELSER.
 
+[[index-multi-value-sparse-vectors]]
+==== Multi-value sparse vectors
+
+When passing in arrays of values for sparse vectors the max value for similarly named features is selected.
+
+The paper Adapting Learned Sparse Retrieval for Long Documents (https://arxiv.org/pdf/2305.18494.pdf) discusses this in more detail.
+In summary, research findings support representation aggregation typically outperforming score aggregation.
+
+For instances where you want to have overlapping feature names use should store them separately or use nested fields.
+
+Below is an example of passing in a document with overlapping feature names.
+Consider that in this example two categories exist for positive sentiment and negative sentiment.
+However, for the purposes of retrieval we also want the overall impact rather than specific sentiment.
+In the example `impact` is stored as a multi-value sparse vector and only the max values of overlapping names are stored.
+More specifically the final `GET` query here returns a `_score` of ~1.2 (which is the `max(impact.delicious[0], impact.delicious[1]` and is approximate because we have a relative error of 0.4% as explained below)
```

Comment:
```suggestion
More specifically the final `GET` query here returns a `_score` of ~1.2 (which is the `max(impact.delicious[0], impact.delicious[1])` and is approximate because we have a relative error of 0.4% as explained below)
```